ocs_ci/ocs/resources/pod.py

```diff
@@ -205,7 +205,12 @@ def workload_setup(self, storage_type, jobs=1):
         """
         work_load = 'fio'
         name = f'test_workload_{work_load}'
-        path = self.get_mount_path()
+        if storage_type == 'block':
```

Comment:
I think it would be better to change the above function name get_mount_path() to get_storage_path() and move lines 208-210 to there

Reply:
@shreekarSS any update on this?

Reply:
Yes @RazTamir , testing the changes and will update asap

Reply:
Updated!